src/LogzioShipper/logzio_shipper.py

```diff
@@ -0,0 +1,137 @@
+import logging
+import requests
+import gzip
+
+from typing import Any
+from requests.adapters import HTTPAdapter, RetryError
+from requests.sessions import InvalidSchema, Session
+from urllib3.util.retry import Retry
+
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+VERSION = "1.0.0"
+
+
+class LogzioShipper:
+    MAX_BODY_SIZE_BYTES = 10 * 1024 * 1024              # 10 MB
+    MAX_BULK_SIZE_BYTES = MAX_BODY_SIZE_BYTES / 10      # 1 MB
+    MAX_LOG_SIZE_BYTES = 500 * 1000                     # 500 KB
+
+    MAX_RETRIES = 3
+    BACKOFF_FACTOR = 1
+    STATUS_FORCELIST = [500, 502, 503, 504]
+    CONNECTION_TIMEOUT_SECONDS = 5
+
+    def __init__(self, logzio_url: str, token: str) -> None:
+        self.logzio_url = "{0}/?token={1}&type=azure".format(logzio_url, token)
```

Comment:
lets change the type to be azure_blob_trigger

Reply:
Done